src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2482,7 +2473,8 @@ NetworkOPsImp::recvValidation(
     JLOG(m_journal.debug())
         << "recvValidation " << val->getLedgerHash() << " from " << source;
     pubValidation(val);
-    return handleNewValidation(app_, val, source);
+    return handleNewValidation(
+        app_, val, source, app_.config().RELAY_UNTRUSTED_VALIDATIONS);
```

Comment:
Consider not passing `RELAY_UNTRUSTED_VALIDATIONS` and add it to the expression in `PeerImp::checkValidation()`: 

> if (app_.getOPs().recvValidation(val, std::to_string(id())) ||
            cluster() || app_.config().RELAY_UNTRUSTED_VALIDATIONS)
    {
            auto const suppression =
                sha512Half(makeSlice(val->getSerialized()));
            overlay_.relay(*packet, suppression);
    }